src/dodal/devices/motors.py

```diff
@@ -2,60 +2,173 @@
 from ophyd_async.epics.motor import Motor
 
 
-class XYZPositioner(StandardReadable):
+class XThetaStage(StandardReadable):
     """
 
-    Standard ophyd_async xyz motor stage, by combining 3 Motors,
-    with added infix for extra flexibliy to allow different axes other than x,y,z.
+    ophyd_async x theta motor stage, combining 2 Motors: a horizontal stage and a
+    rotational stage mutually perpendicular to each other and the beam.
 
     Parameters
     ----------
     prefix:
-        EPICS PV (Common part up to and including :).
+        Common part of the EPICS PV for all motors, including ":".
     name:
-        name for the stage.
-    infix:
-        EPICS PV, default is the ("X", "Y", "Z").
-    Notes
-    -----
-    Example usage::
-        async with init_devices():
-            xyz_stage = XYZPositioner("BLXX-MO-STAGE-XX:")
-    Or::
-        with init_devices():
-            xyz_stage = XYZPositioner("BLXX-MO-STAGE-XX:", infix = ("A", "B", "C"))
+        Name of the stage, each child motor will be named "{name}-{field_name}"
+    *_infix:
+        Infix between the common prefix and the EPICS motor record fields for the field.
+
+    """
+
+    def __init__(
+        self, prefix: str, name: str = "", x_infix: str = "X", theta_infix: str = "A"
+    ):
+        with self.add_children_as_readables():
+            self.x = Motor(prefix + x_infix)
+            self.theta = Motor(prefix + theta_infix)
+        super().__init__(name=name)
+
+
+class XYStage(StandardReadable):
+    """
+
+    ophyd_async x y motor stage, combining 2 Motors, mutually perpendicular to the beam.
```

Comment:
Nit: Coordinate systems are horrible beasts. Some issues with defining it here:
* You haven't defined it in any of the other generic devices, we should probably do it in all of them or none
* There are very poor standards on coordinate systems in general, I am sure someone is using y to be parallel to the beam (and z to be vertical) somewhere
* Some devices have moving frames of reference compared to the beam. For example, our gonio has x,y as perpendicular to the beam when it is at an omega of 0 and y parallel to the beam when at omega 90

I would suggest maybe changing to something like "Usually defined as x and y perpendicular to the beam and y vertical in lab space".